app/src/main/java/com/puzzlebench/clean_marvel_kotlin/presentation/adapter/viewholder/viewholder.kt

```diff
@@ -12,7 +12,7 @@ class CharactersAdapterViewHolder(view: View, val listener: CharacterListener) :
 
     fun bind(item: Character) = with(itemView) {
         tv_item.text = item.name
-        val string = item.thumbnail.path + "." + item.thumbnail.extension
+        val string = item.thumbnail?.path + "." + item.thumbnail?.extension
```

Comment:
please use the kotlin style concat

"${item.thumbnail?.path}$DOT${item.thumbnail?.extension}"

Reply:
done